js/modules/boost/wgl-renderer.js

```diff
@@ -602,6 +620,28 @@ function GLRenderer(postRenderCallback) {
                 continue;
             }
 
+            // Note: Boost requires that zones are sorted!
+            if (zones) {
+                pcolor = zoneDefColor.rgba;
```

Comment:
@cvasseng
I am getting this bug with this line: [example](https://jsfiddle.net/8wujdtx0/)
The curve should show purple up to a y value of 3, red up to a value of 5, and **zoneDefColor** till infinity (Line 285).
But I think this happens because you mutate the zoneDefColor.rgba when looping through each point, so the rgba is normalized every time and rgb becomes [0, 0, 0] which is black.

Assuming no performance impact, we can simply clone the rgb for each point e.g.
Line 625:  `pcolor = zoneDefColor.rgba.slice()`